File: articles/algorithmic-risk-management-and-ai-governance/java/GovernanceReadinessScore.java
public class GovernanceReadinessScore {
    public static void main(String[] args) {
        double[] scores = {0.60, 0.62, 0.58, 0.52, 0.46, 0.50};
        double total = 0.0;
        for (double score : scores) total += score;
        System.out.printf("governance_readiness_score=%.4f%n", total / scores.length);
    }
}

File: articles/algorithmic-risk-management-and-ai-governance/go/governance_readiness_score.go
package main

import "fmt"

func main() {
	scores := []float64{0.60, 0.62, 0.58, 0.52, 0.46, 0.50}
	total := 0.0
	for _, score := range scores {
		total += score
	}
	fmt.Printf("governance_readiness_score=%.4f\n", total/float64(len(scores)))
}
